apps/happy-blocks/src/support-page/support-page-embed.tsx

```diff
@@ -0,0 +1,60 @@
+import { __, sprintf } from '@wordpress/i18n';
+import { SupportPageBlockAttributes } from './block';
+import { WordPressIcon } from './icon';
+import { InlineSkeleton } from './inline-skeleton';
+
+/**
+ * Rendered embed for the Support Page block.
+ */
+export const SupportPageEmbed = ( props: { attributes: SupportPageBlockAttributes } ) => {
+	const loaded = !! props.attributes.content;
```

Comment:
What do you think about checking content's length, whether it's greater than, say, 10 instead of `!!`? Just in case we load something but it's not a real content.

Reply:
Probably, that check could land in the settings placeholder, when parsing the URL for the first time. To let the author know that he has used an incorrect page URL.

However, I don't feel like it is necessary. As long as the page content is returned by WP API, it can be embedded. And we show a preview before publishing the post. So if the content is not what's expected by the author, he can replace the URL.